vnext/Desktop.DLL/react-native-win32.x64.def

```diff
@@ -191,3 +191,7 @@ YGUnitToString
 
 DllCanUnloadNow = WINRT_CanUnloadNow                    PRIVATE
 DllGetActivationFactory = WINRT_GetActivationFactory    PRIVATE
+
+;TODO: Find another way to use in Desktop.IntegrationTests
```

Comment:
For reviewers:

`React.Windows.Desktop.IntegrationTests` now consumes `JSValueObject`.
This creates an unresolved symbol issue (they are defined inside the Desktop DLL, but not explicitly exported).

Is there a way around this?

Reply:
JSValueObject is part of MS.RN.Cxx, which the integration tests should probably link against.  It shouldn't be exported from the DLL - it is not ABI safe.